admin/app/controllers/TroubleshooterController.scala

```diff
@@ -34,10 +34,12 @@ class TroubleshooterController(wsClient: WSClient)(implicit appContext: Applicat
   val contentApi = new ContentApiClient(capiHttpClient)
   val previewContentApi = new PreviewContentApi(capiHttpClient)
 
-  private lazy val awsEc2Client: Option[AmazonEC2Client] = credentials.map { credentials =>
-    val client = new AmazonEC2Client(credentials)
-    client.setRegion(Region.getRegion(Regions.EU_WEST_1))
-    client
+  private lazy val awsEc2Client: Option[AmazonEC2] = credentials.map { credentials =>
+    AmazonEC2ClientBuilder
+      .standard()
+      .withCredentials(credentials)
+      .withRegion(conf.Configuration.aws.region)
```

Comment:
Yesterday, I discover Regions.getCurrentRegion() if you run it from an EC2 instance it will populate the region for you.
Maybe it helps for future references.